### PR TITLE
orderer: fix function names

### PR DESCRIPTION
**Pull request by knowmost**
#### Type of change

- Documentation update

#### Description

 fix function names

### CODE REVIEW

File: orderer/consensus/smartbft/signer.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func (s *Signer) SignProposal(proposal types.Proposal, _ []byte) *types.Signatur
 	}
 }
 
-// NewSignatureHeader creates a SignatureHeader with the correct signing identity and a valid nonce
+// newIdentifierHeaderOrPanic creates a SignatureHeader with the correct signing identity and a valid nonce
```
Comment (satota2) [inline orderer/consensus/smartbft/signer.go]:
```suggestion
// newIdentifierHeaderOrPanic creates an IdentifierHeader with the signer's identifier and a valid nonce
```
The descriptions should also be modified to match the actual function behavior.

**Review by satota2 — CHANGES_REQUESTED** [review]
@knowmost Thank you for your contribution.
I found a minor modification and commented on it, please check it out.

File: orderer/consensus/smartbft/signer.go (hunk 1 above)
Comment (knowmost) [reply orderer/consensus/smartbft/signer.go]: Thanks!  Modified!

**Review by satota2 — APPROVED** [review]
@knowmost Thank you for your update. LGTM!

**Pull request merged**